lib/cache.rb

```diff
@@ -0,0 +1,29 @@
+require 'dalli'
+require 'resque'
+require_relative '../jobs/reload_cache'
+
+module Lib
+    module Cache
+        
+        # Create a static Dalli object to handle Memcache connections.
+        DC = Dalli::Client.new(ENV['MEMCACHIER_SERVERS'], { :namespace => ENV['MEMCACHIER_NAMESPACE'], :compress => true })
```

Comment:
The Dalli documentation claims that we don't need to pass in a username and a password if we are on Heroku. So if something breaks in the future with caching, start looking here.